.builders/scripts/utils.py

```diff
@@ -32,3 +39,112 @@ def extract_metadata(wheel: Path) -> email.Message:
             raise RuntimeError(message) from None
 
     return email.message_from_string(metadata_file_contents)
+
+
+
+def iter_wheels(source_dir: str) -> Iterator[Path]:
+    for entry in sorted(Path(source_dir).iterdir(), key=lambda entry: entry.name.casefold()):
+        if entry.suffix == '.whl' and entry.is_file():
+            yield entry
+
+def remove_test_files(wheel_path: Path) -> None:
```

Comment:
Let's keep the `utils` module for only code that's shared between `build_wheels.py` and `repair_wheels.py`. This means we move this function closer to where it's used. Same goes for is_excluded_from_wheel